test/uk/gov/hmrc/ngrnotify/backend/controllers/PhysicalControllerSpec.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ngrnotify.backend.controllers
+
+import org.apache.pekko.stream.Materializer
+import org.apache.pekko.stream.testkit.NoMaterializer
+import play.api.Application
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.Json
+import play.api.test.Helpers.*
+import play.api.test.{FakeRequest, Helpers}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, StringContextOps}
+import uk.gov.hmrc.ngrnotify.backend.base.AnyWordAppSpec
+import uk.gov.hmrc.ngrnotify.backend.testUtils.RequestBuilderStub
+import uk.gov.hmrc.ngrnotify.controllers.PhysicalController
+import uk.gov.hmrc.ngrnotify.model.propertyDetails.{AnythingElseData, ChangeToUseOfSpace, CredId, PropertyChangesRequest}
+
+
+import java.time.LocalDate
+import scala.collection.immutable.Seq
+
+class PhysicalControllerSpec extends AnyWordAppSpec:
+
+  private val controller = inject[PhysicalController]
+
+  given Materializer = NoMaterializer
+
+  override def fakeApplication(): Application =
+    val httpClientV2Mock = mock[HttpClientV2]
+
+    when(
+      httpClientV2Mock.post(eqTo(url"http://localhost:1501/job/physical"))(using any[HeaderCarrier])
```

Comment:
You may also consider implementing a stub endpoint in ngr-stub and adding it to the ngr-notify configuration, similar to the other HIP endpoints.
https://github.com/hmrc/ngr-notify/blob/main/conf/application.conf#L77-L78
https://github.com/hmrc/ngr-ui-tests/blob/main/src/test/resources/stubData/hipGerRatepayer.json